data_steward/utils/participant_summary_requests.py

```diff
@@ -65,16 +66,34 @@ def get_participant_data(url, headers):
     return participant_data
 
 
-def get_deactivated_participants(project_id, columns):
+def get_deactivated_participants(project_id, dataset_id, tablename, columns):
     """
     Fetches all deactivated participants via API if suspensionStatus = 'NO_CONTACT'
+    and stores all the deactivated participants in a BigQuery dataset table
 
-    :param project_id: THE RDR project that contains participant summary data
-    :param columns: columns to be pushed to a table in BigQuery
+    :param project_id: The RDR project that contains participant summary data
+    :param dataset_id: The dataset name
+    :param tablename: The name of the table to house the deactivated participant data
+    :param columns: columns to be pushed to a table in BigQuery in the form of a list of strings
 
     :return: returns dataframe of deactivated participants
     """
 
+    # Parameter checks
+    if not project_id or not isinstance(project_id, str):
+        raise RuntimeError(f'Please specify the RDR project')
+
+    if not dataset_id or not isinstance(dataset_id, str):
+        raise RuntimeError(f'Please provide a dataset_id')
+
+    if not tablename or not isinstance(tablename, str):
+        raise RuntimeError(
+            f'Please provide a tablename to house deactivated participant data')
+
+    if not columns or not isinstance(columns, list):
+        raise RuntimeError(
+            'Please provide a list of columns to be pushed to BigQuery table')
```

Comment:
The `isinstance` check can accommodate the presence of `None`.  So, these could all be simplified to:

```
if not isinstance(variable, type):
```

Reply:
I simplified all the parameter check statements